src/group-configurations/index.jsx

```diff
@@ -86,7 +88,9 @@ const GroupConfigurations = ({ courseId }) => {
               {shouldShowExperimentGroups && (
                 <ExperimentConfigurationsSection
                   availableGroups={experimentGroupConfigurations}
-                  experimentConfigurationActions={experimentConfigurationActions}
+                  experimentConfigurationActions={
+                    experimentConfigurationActions
+                  }
```

Comment:
[clarify]: Why did you change the formatting?)

```suggestion
                  experimentConfigurationActions={experimentConfigurationActions}
```

Reply:
ok, leave it as it was